measureme/Cargo.toml

```diff
@@ -18,3 +18,6 @@ rustc-hash = "1.0.1"
 
 [target.'cfg(not(target_arch="wasm32"))'.dependencies]
 memmap = "0.6.0"
+
+[dev-dependencies]
+tools_lib = { path = "../tools_lib" }
```

Comment:
I'd like to get rid of this (pseudo)circular dependency. I think it's fine to move the end-to-end serialization tests into `tools_lib`.